easybuild/easyconfigs/t/tbb/tbb-2020.3-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,19 @@
+name = 'tbb'
+version = '2020.3'
```

Comment:
@sassy-crick What is the reason to use an old version of `tbb`? Is it the compatibility problem between classic and oneAPI `tbb`? I have opened the new one in https://github.com/easybuilders/easybuild-easyconfigs/pull/14366 In case you need the latest classic `tbb` I would suggest to go with v2021.3

Reply:
Unfortunately, there is. I have tried the same version as you suggested and I got:
```
In file included from /home/sassy/apps/build/UShER/0.5.0/gompi-2021b/usher-0.5.0/src/matOptimize/transpose_vcf/transposed_vcf_print_name.cpp:1:
/home/sassy/apps/build/UShER/0.5.0/gompi-2021b/usher-0.5.0/src/matOptimize/transpose_vcf/transpose_vcf.hpp:2:10: fatal error: tbb/pipeline.h: No such file or directory
    2 | #include <tbb/pipeline.h>
      |          ^~~~~~~~~~~~~~~~
compilation terminated.
```
A bit more digging:
```
$ ls
2020.3-GCCcore-10.2.0  2020.3-GCCcore-10.3.0  2020.3-GCCcore-11.2.0  2021.1.1-GCCcore-10.3.0  2021.2.0-GCCcore-10.3.0  2021.3.0-GCCcore-10.3.0  2021.4.0-GCCcore-11.2.0
$ find ./ -iname pipeline.h
./2020.3-GCCcore-11.2.0/include/tbb/pipeline.h
./2020.3-GCCcore-10.3.0/include/tbb/pipeline.h
./2020.3-GCCcore-10.2.0/include/tbb/pipeline.h
```
So, for me it looks like that [UShER](https://github.com/easybuilders/easybuild-easyconfigs/pull/14305) requires these header files and thus the old version. Not what I like to be frank.
That said: if you come with a solution for using the newer version I am all up for it.

Reply:
I guess it doesn't hurt to have this old version in a new toolchain, even though we may need to make an exception in the easyconfig tests if/when multiple versions are used as dependency in the same toolchain version

Reply:
Yes, I agree. This is a good reason to have this old version in the new toolchain. I think the exception should be added in the PR that uses the older version as dependency.

Reply:
How would I add the exception to [UShER](https://github.com/easybuilders/easybuild-easyconfigs/pull/14305) or is that something only the maintainer can do?

Reply:
@sassy-crick you can do it yourself, simply add the exception to `check_dep_vars` in `easybuild-easyconfigs/test/easyconfigs/easyconfigs.py` and update the PR

note that the tests won't fail now, because the new version isn't used as a dependency yet, but if I understood what @SebastianAchilles meant, the exception should be added in the UShER PR anyway because that's the one that needs the older dependency